src/render.tsx

```diff
@@ -100,10 +101,22 @@ function buildRenderResult(
     ...getQueriesForElement(instance),
     update,
     unmount,
-    container: instance,
     rerender: update, // alias for `update`
     toJSON: renderer.toJSON,
     debug: debug(instance, renderer),
+    get root() {
+      return getHostChildren(instance)[0];
+    },
+    UNSAFE_root: instance,
+    get container() {
+      if (!getConfig().useBreakingChanges) {
+        return instance;
+      }
+
+      // eslint-disable-next-line no-console
+      console.warn(`'container' property has been renamed to 'UNSAFE_root'`);
+      throw new Error("'container' property has been renamed to 'UNSAFE_root'");
```

Comment:
Why warn and error at the same time? Error is descriptive enough imho

Reply:
remove `console.warn`?

Reply:
Yeah